### PR TITLE
Add CI with Github Actions

### PR DESCRIPTION
**Pull request by nvgoldin**
On each commit to master/develop:
1. Run `black` with `--check`, will fail if not all files are formatted.
2. Run unit tests, will fail the pipeline if any of them fails.

@asaf-kali note one of the unit tests failed - I fixed it - not sure what was the reason (it happens on `develop` too I assume).

Closes https://github.com/asaf-kali/coursist/issues/118#issue-654358143

### CODE REVIEW

File: academic_helper/tests/test_shnaton_parser.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def test_simple_base_models(self):
         group = groups[0]
 
         classes = CourseClass.objects.filter(group=group)
-        self.assertEquals(len(classes), 14, "There are 14 classes")
+        self.assertEquals(len(classes), 13, "There are 13 classes")
```
Comment (asaf-kali) [inline academic_helper/tests/test_shnaton_parser.py:46]: This is not good! It covers up a real bug, not related to this PR.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      with:
+        python-version: 3.8
+    - name: Install python dependencies
+      run: |
+        pip install -r requirements-dev.txt
+        pip install -r requirements.txt
+    - name: Black Code Formatter
+      uses: lgeiger/black-action@v1.0.1
+      with:
+        args: ". --check -l 120"
```
Comment (asaf-kali) [inline .github/workflows/main.yml:30]: There is no need to run black on the CI. What we need is to make sure black doesn't change any file (thus making sure the code on the branch is already black-formatted).

File: academic_helper/tests/test_shnaton_parser.py (hunk 1 above)
Comment (nvgoldin) [reply academic_helper/tests/test_shnaton_parser.py:46]: wops, sure I'll revert. I think its recent, couple of days ago tests were running for me.

Comment (nvgoldin) [reply academic_helper/tests/test_shnaton_parser.py]: reverted

**Pull request merged**